crates/interpreter/src/interpreter_action/call_inputs.rs

```diff
@@ -36,7 +36,7 @@ pub struct CallInputs {
     ///
     /// Previously `context.scheme`.
     pub scheme: CallScheme,
-    /// Whether the call is initiated inside a static call.
+    /// Whether the call is, or is initiated inside a static call.
```

Comment:
think a word is missing here

Reply:
is,
or
is initialized inside

I think this is correct

Reply:
rewrote it anyway

Reply:
ah i see what this means now, yeah i think it's correct but it's hard to parse 😅